automl/automl-library/library.whizzml

```diff
@@ -242,29 +266,33 @@
           (lambda (i) (_flatline-item (items i) (fields ((items i) "field_id"))))
           lhs (map get-item-fn (rule "lhs"))
           rhs (map get-item-fn (rule "rhs"))
+          metric (rule "leverage" 1)
```

Comment:
I thought you also used rules based on lift, but maybe you don't

Reply:
I have the problem I mentioned previously with lift. It doesn't take a fix range of values